docs/guides/Checkout/creating-a-regular-order-from-an-existing-cart.mdx

```diff
@@ -0,0 +1,684 @@
+---
+title: "Creating a regular order from an existing cart"
+slug: "creating-a-regular-order-from-an-existing-cart"
+hidden: false
+createdAt: "2023-07-18T17:08:52.219Z"
+updatedAt: "2023-07-18T17:08:52.219Z"
+excerpt: "Learn how to use VTEX APIs to place a regular order from an existing cart."
+---
+
+This step-by-step guide will walk you through the process of using the [Checkout API](https://developers.vtex.com/docs/api-reference/checkout-api) to create an order from an existing shopping cart. This process is divided into the following steps:
+
+1. **Get cart information:** The process begins with retrieving the `orderFormId` of the current shopping cart via the [Get current or create a new cart](https://developers.vtex.com/docs/api-reference/checkout-api#get-/api/checkout/pub/orderForm) endpoint.
+2. **Place the order:** The process continues by placing the order on the VTEX platform via the [Place order from an existing cart](https://developers.vtex.com/docs/api-reference/checkout-api#post-/api/checkout/pub/orderForm/-orderFormId-/transaction) endpoint.
+3. **Handling the order payment and processing:** The process ends by sending the necessary payment data to resolve the order payment and processing order details.
+
+## Before you begin
+
+Before proceeding any further, ensure that you have a valid `appKey` and `appToken` granted with the necessary permissions to access the [Checkout](https://developers.vtex.com/docs/api-reference/checkout-api) and [Payments Gateway](https://developers.vtex.com/docs/api-reference/payments-gateway-api) APIs. For more information, please refer to the [Roles](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc) article.
+
+> ✅ Click the steps below to navigate through this article and explore specific topics. Relevant code will be highlighted and focused for your convenience.
```

Comment:
Aqui eu sugiro o mesmo _disclaimer_ do outro PR.
"Click the steps below to navigate through this article and explore specific topics. The relevant codes for each phase or parameter will be highlighted on the right side of the screen".